src/content/1.7/development/components/grid/tutorials/work-with-search-form/in-1-7-5.md

```diff
@@ -0,0 +1,237 @@
+---
+title: How to work with the Search Form in 1.7.5
+menuTitle: In 1.7.5
+weight: 2
+---
+
+# How to work with the Search Form
+
+The grid component allows to filter its content, to manage the filtering you will need to define the following elements:
+
+- `GridDefinition::getFilters`
+- `Filters` class
+- Controllers:
+  - Search action
+  - List action
+  - Reset filter action
+
+## Add Filters into GridDefinition
+
+```php
+final class EmailLogsDefinitionFactory extends AbstractGridDefinitionFactory {
```

Comment:
This doesnt bother me that much, but your bracket should be positionned in the next line :)

Reply:
Yep nice catch